cypress/e2e/gosub.ts

```diff
@@ -24,7 +24,7 @@ describe('Subroutine workflow block', () => {
                 "workflowId": "madeUpFlowIDA"
             }
         ]);
-        cy.get('mat-toolbar > button mat-icon').contains('settings').click();
+        cy.get('mat-toolbar > button mat-icon').contains('settings').click({force: true});
```

Comment:
Why do we need this `force: true`?

Reply:
We may not with the new Cypress version! I need to check.